### PR TITLE
✨ Nhj strelka2fields docker

### PR DESCRIPTION
**Pull request by NHJohnson**
## Description

This would represent a Dockerfile and circleci config updates for a script that adds the missing fields to Strelka2 VCFs.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Dockerfile builds on EC2 instance
- [X] Tested on EC2 with this input VCF: https://cavatica.sbgenomics.com/u/zhangb1/kf-somatic-tools-test/files/5ffdf91ce4b0a6d32f329e6b/   tumor_name  BS_F0GNWEJJ      normal_name BS_F5EX3ZY5

**Test Configuration**:
* Environment:
* Test files:
* VCF: https://cavatica.sbgenomics.com/u/zhangb1/kf-somatic-tools-test/files/5ffdf91ce4b0a6d32f329e6b/'
* tumor_name  BS_F0GNWEJJ      
* normal_name BS_F5EX3ZY5

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [X] I have committed any related changes to the PR

### CODE REVIEW

File: add-strelka2-fields/1.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    SAMTOOLS_VERSION=1.9 \
+    PYSAM_VERSION=0.16.0
+RUN conda create -qy -p /usr/local \
+        -c conda-forge \
+        -c bioconda \
+        -c defaults \
+        htslib==${HTSLIB_VERSION} \
+        samtools==${SAMTOOLS_VERSION} \
+        pysam==${PYSAM_VERSION} \
+        && curl -sL -o /usr/bin/add_strelka2_fields.py https://raw.githubusercontent.com/kids-first/kf-somatic-workflow/master/scripts/add_strelka2_fields.py
```
Comment (dmiller15) [inline add-strelka2-fields/1.0.0/Dockerfile:23]: Dubious pulling from master. I'd rather there be a release or particular commit you can pull.

Comment (migbro) [reply add-strelka2-fields/1.0.0/Dockerfile:23]: Agreed, although it's a bit "check and egg" if the release doesn't exist yet. However, if we already have approved that script, then you can def cut a release using semantic versioning, then be sure to point to the release version of the script instead. Does that make sense?

File: add-strelka2-fields/1.0.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
```
Comment (migbro) [inline add-strelka2-fields/1.0.0/Dockerfile:24]: I'd also remove software and files not needed. For instance, I noticed that conda keeps all these source `.tar.bz2` files that aren't needed after build, so:
```
conda clean -a -y && \
        apt remove curl -y && apt autoclean -y
```
should cover that

File: add-strelka2-fields/1.0.0/Dockerfile (hunk 1 above)
Comment (dmiller15) [reply add-strelka2-fields/1.0.0/Dockerfile:23]: It doesn't need to be chicken/egg. He can build this image locally and push to a private registry then test using that to get to a version/commit that he views as stable. If we're lagging on releases he can always use a commit hash to pull a specific version of the file.

Comment (NHJohnson) [reply add-strelka2-fields/1.0.0/Dockerfile]: I changed this to refer to a particular commit.

File: add-strelka2-fields/1.0.0/Dockerfile (hunk 2 above)
Comment (NHJohnson) [reply add-strelka2-fields/1.0.0/Dockerfile:26]: Sounds good, thanks!

**Review by migbro — APPROVED** [review]
builds and runs!

**Review by dmiller15 — APPROVED** [review]

**Pull request merged**